### PR TITLE
kodi: bump to 17.0-beta6-654f1b7

### PR DESCRIPTION
**Pull request by MilhouseVH**
IMX6: I've had to remove projects/imx6/patches/libcec/libcec-100-from-xbian.patch as no longer compatible with libcec-4.0.0.

I've not bumped inputstream.mpd and inputstream.smoothstream as they are obsolete - will be removed by #994.

LibreELEC branding is now included in Estuary v2. This does make it a little more difficult for custom distro naming as we lose `@DISTRONAME@` support. Minor issue though.

### CODE REVIEW

File: packages/mediacenter/kodi/patches/aarch64/kodi-0002-AESinkALSA-force-default-device-for-non-passthrough.patch [hunk 1]
```diff
@@ -11,14 +11,14 @@ diff --git a/xbmc/cores/AudioEngine/Sinks/AESinkALSA.cpp b/xbmc/cores/AudioEngin
...
+@@ -535,8 +535,8 @@ bool CAESinkALSA::Initialize(AEAudioFormat &format, std::string &device)
      m_passthrough   = false;
    }
  #if defined(HAS_LIBAMCODEC)
 -  if (aml_present())
 +  if (!m_passthrough && device.find("hdmi:CARD=AMLM8AUDIO") != std::string::npos)
    {
 -    aml_set_audio_passthrough(m_passthrough);
-     device = "default";
++    device = "default";
```
Comment (Kwiboo) [inline packages/mediacenter/kodi/patches/aarch64/kodi-0002-AESinkALSA-force-default-device-for-non-passthrough.patch]: This is wrong and will stop USB-sound from working again, `device = "default"` was removed from upstream and from the previous patch in LE.

**Comment by kszaq** [comment]
@Kwiboo You are right, this was a late night patch. I will push a fix shorty.

**Comment by Kwiboo** [comment]
@kszaq I might have been wrong myself, don't we need to set it to default to force device 0 for non-passthrough?

**Comment by kszaq** [comment]
PCM audio might not work if forwarded to `hdmi:CARD=AMLM8AUDIO,DEV=0` so it could be a good idea to force it to default device whenever user selects HDMI for PCM output.

**Comment by Kwiboo** [comment]
You are correct, this is good as is, sorry for the noise

**Pull request merged**